### PR TITLE
show feed only for activity feed tab

### PR DESCRIPTION
**Pull request by ericcf**

### CODE REVIEW

File: app/views/social_networking/homes/show.html.erb [hunk 1]
```diff
@@ -68,7 +68,8 @@
     id="infinite-feed"
     infinite-scroll="home.getMore()"
     infinite-scroll-immediate-check="true"
-    infinite-scroll-disabled="false">
+    infinite-scroll-disabled="false"
+    style="display: none;">
```
Comment (michaelwehrley) [inline app/views/social_networking/homes/show.html.erb]: Listen to https://fivejs.codeschool.com/episodes/91-episode-84-july-16th-2015 from 1:10-1:45.  They recommend a better practice (now specifically in the context of jQuery 3, but the argument could be made that it is better practice regardless) of using classes and not `display: none;`

File: spec/javascripts/content_providers/feed_tool_provider_spec.js [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+//= require content_providers/feed_tool_provider
+
+'use strict';
+
+window.SUNNYSIDE_GLOBALS = {
+  ANIMATION_EASE_DELAY_MS: 0
+};
+
+describe('feed tool provider', function() {
+  function getFixture() {
```
Comment (michaelwehrley) [inline spec/javascripts/content_providers/feed_tool_provider_spec.js]: :+1:

File: app/views/social_networking/homes/show.html.erb (hunk 1 above)
Comment (ericcf) [inline app/views/social_networking/homes/show.html.erb]: I missed the part where they explain how to use `addClass` and `removeClass` to animate as we're doing with `show` and `hide` ;-)

File: spec/javascripts/content_providers/feed_tool_provider_spec.js [hunk 3]
```diff
@@ -0,0 +1,246 @@
...
+  });
+
+  describe('the communal plot button listener', function() {
+    function initCommunalPlotButton() {
+      return getFixture().empty()
+                         .append('<div id="communal-plot-btn" />')
+                         .find('#communal-plot-btn');
+    }
+
+    it('removes the home toggle active class', function() {
```
Comment (michaelwehrley) [inline spec/javascripts/content_providers/feed_tool_provider_spec.js]: lots of conditions to cover here.  Looks good!

File: app/views/social_networking/homes/show.html.erb (hunk 1 above)
Comment (michaelwehrley) [inline app/views/social_networking/homes/show.html.erb]: Not sure I follow?  For future reference, if the object is initially hidden, as it is above, using show and hide is not the recommend practice.

**Comment by michaelwehrley** [comment]
:+1:

**Pull request merged**